viewer/viewer/templates/viewer/page_list.html

```diff
@@ -120,4 +121,86 @@ <h4>
+  </div>
+  <ul class="m-list">
+    <!-- prettier-ignore -->
+    <li>
+      Download
+      <a
+        class="m-list_link a-link a-link__icon"
+        href="{% url 'errors' %}?format=csv">
+        <span class="a-link_text">website errors (including 404s)</span>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 12 19"
+          class="cf-icon-svg">
+          <path
+            d="M11.16 16.153a.477.477 0 0 1-.476.475H1.316a.476.476 0 0 1-.475-.475V3.046a.476.476 0 0 1 .475-.475h6.95l2.893 2.893zm-1.11-9.925H8.059a.575.575 0 0 1-.574-.573V3.679H1.95v11.84h8.102zm-1.234 5.604L6.388 14.26a.554.554 0 0 1-.784 0l-2.428-2.428a.554.554 0 1 1 .783-.784l1.483 1.482V7.41a.554.554 0 1 1 1.108 0v5.12l1.482-1.482a.554.554 0 0 1 .784.783z"></path>
+        </svg>
+      </a>
+      and
+      <a
+        class="m-list_link a-link a-link__icon"
+        href="{% url 'redirects' %}?format=csv">
+        <span class="a-link_text">website redirects</span>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 12 19"
+          class="cf-icon-svg">
+          <path
+            d="M11.16 16.153a.477.477 0 0 1-.476.475H1.316a.476.476 0 0 1-.475-.475V3.046a.476.476 0 0 1 .475-.475h6.95l2.893 2.893zm-1.11-9.925H8.059a.575.575 0 0 1-.574-.573V3.679H1.95v11.84h8.102zm-1.234 5.604L6.388 14.26a.554.554 0 0 1-.784 0l-2.428-2.428a.554.554 0 1 1 .783-.784l1.483 1.482V7.41a.554.554 0 1 1 1.108 0v5.12l1.482-1.482a.554.554 0 0 1 .784.783z"></path></svg></a
+      > (CSV files)
+    </li>
+    <li>
+      Explore the data via API:
+      <a class="m-list_link a-link" href="{% url 'index' %}?format=api">pages</a
+      >,
+      <a class="m-list_link a-link" href="{% url 'components' %}?format=api"
+        >components</a
+      >,
+      <a class="m-list_link a-link" href="{% url 'errors' %}?format=api"
+        >errors</a
+      >, and
+      <a class="m-list_link a-link" href="{% url 'redirects' %}?format=api"
+        >redirects</a
+      >
+    </li>
+    <li class="m-list_item">
+      <a
+        class="m-list_link a-link a-link__icon"
+        href="{% url 'download-database' %}">
+        <span class="a-link_text">Download the raw SQLite database</span>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 12 19"
+          class="cf-icon-svg">
+          <path
+            d="M11.16 16.153a.477.477 0 0 1-.476.475H1.316a.476.476 0 0 1-.475-.475V3.046a.476.476 0 0 1 .475-.475h6.95l2.893 2.893zm-1.11-9.925H8.059a.575.575 0 0 1-.574-.573V3.679H1.95v11.84h8.102zm-1.234 5.604L6.388 14.26a.554.554 0 0 1-.784 0l-2.428-2.428a.554.554 0 1 1 .783-.784l1.483 1.482V7.41a.554.554 0 1 1 1.108 0v5.12l1.482-1.482a.554.554 0 0 1 .784.783z"></path>
+        </svg>
+      </a>
+      ({{ crawl_stats.database_size | filesizeformat }}) to
+      <a
+        class="m-list_link a-link a-link__icon"
+        href="https://github.com/cfpb/crawsqueal#how-to-query-the-crawler-database">
```

Comment:
I forgot that I changed this documentation but neglected to update this link, if you don't mind adding this to your PR:

```suggestion
        href="https://github.com/cfpb/crawsqueal#searching-the-crawl-database">
```